index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WebDev Summit </title>
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+        <!-- Navigation Section -->
+        <header>
+            <div class="contact-us-navbar">
+                <ul class="contact-us-navbar-ul">
+                    <li class="fa fa-facebook contact-us-navbar-li"></li>
+                    <li class="fa fa-twitter contact-us-navbar-li"></li>
+                    <li class="contact-us-navbar-li">English</li>
+                    <li class="contact-us-navbar-li">My page</li>
+                    <li class="contact-us-navbar-li">Logout</li>
+                </ul>
+            </div>
+            <nav>
+                <div class="mobile-menu">
+                    <button class="mobile-menu-button">&#9776;</button>
+                  </div>
+                  <div class="mobile-menu-options">
+                    <button class="mobile-menu-closing-btn">
+                        &times;
+                    </button>
+                    <ul class="mobile-menu-list-ul">
+                        <li class="mobile-menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="mobile-menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                  </div>
+                <div class="logo">
+                    <a href="./index.html"><img src="./assets/imgs/logo.png" alt="Logo"></a>
+                </div>
+                <div class="menu">
+                    <ul class="menu-list-ul">
+                        <li class="menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                </div>
+            </nav>
```

Comment:
**OPTIONAL** - Good job done so far, but your navbar does not look like the design template, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. Please fix this for both pages
Your design:
- Please make sure you have all links present in the design template on your project 👍 👍 
![navbar](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/73607512/68507e06-7ade-45e7-900d-fe10d5a97155)

Design template:
![jrthreth](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/73607512/3ce5c4f0-a0f0-4a86-9683-89389a97d0b5)